stdlib/REPL/test/replcompletions.jl

```diff
@@ -110,7 +110,7 @@ end
 
 let s = "using REP"
     c, r = test_complete(s)
-    @test count(isequal("REPL"), c) == 1
+    @test count(isequal("REPL"), c) == 2
```

Comment:
I have no clue what's going on here...

Reply:
Weird, this now breaks the CI test,  but was needed locally on my machine for tests to pass 🤔.

Reply:
> Weird, this now breaks the CI test, but was needed locally on my machine for tests to pass

Yeah, it's a bug in the REPL tests.

Reply:
Does CI run the tests with different options or why isn't this bug present there?

Reply:
https://github.com/JuliaLang/julia/issues/32377 has some more info.

Reply:
Ah, thanks for the pointer!